src/components/GaugeCard/GaugeCard.jsx

```diff
@@ -0,0 +1,201 @@
+import React, { useState, useEffect } from 'react';
+import classnames from 'classnames';
+
+import { CARD_CONTENT_PADDING } from '../../constants/LayoutConstants';
+import { CardPropTypes, GaugeCardPropTypes } from '../../constants/PropTypes';
+import Card from '../Card/Card';
+import { settings } from '../../constants/Settings';
+
+const { iotPrefix } = settings;
+// r value of the circle in SVG
+const radius = 30;
+// radius doubled plus stroke
+const gaugeSize = radius * 2 + 8;
+export const getStrokeDash = (value = 0) => {
+  // circumference of SVG.
+  const circum = 2 * Math.PI * radius;
+  // length of stroke to match percentage
+  return (value * circum) / 100;
+};
+
+export const getColor = (gauge, value) => {
+  let { color } = gauge;
+  let grade = '';
+  const comparisons = {
+    '<': (a, b) => a < b,
+    '>': (a, b) => a > b,
+    '=': (a, b) => a === b,
+    '<=': (a, b) => a <= b,
+    '>=': (a, b) => a >= b,
+  };
+  if (gauge.thresholds) {
+    gauge.thresholds.forEach(thresh => {
+      if (comparisons[thresh.comparison](value, thresh.value)) {
+        color = thresh.color; /* eslint-disable-line */
+        grade = thresh.label;
+      }
+    });
+    return {
+      grade,
+      color,
+    };
+  }
+  return {
+    grade,
+    color,
+  };
+};
+const GaugeCard = ({
+  id,
+  title,
+  tooltip,
+  content: { gauges },
+  values,
+  data,
+  isLoading,
+  hasMoreData,
+  size,
+  className,
+  ...others
+}) => {
+  const [loadedState, setLoadedState] = useState(false);
+  useEffect(() => {
```

Comment:
I'm confused by this.  What are we doing here?  Seems like we are setting our loaded state to true after 1 second??

Reply:
This is to trigger the load animation of the gauge